tests/system.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-extern crate rustfmt;
+extern crate rustfmt_nightly as rustfmt;
```

Comment:
`cargo test` errored without this. But even with this, there are failing tests for me.